nestkernel/nest_types.h

```diff
@@ -111,8 +111,11 @@ typedef unsigned long ulong_t; ///< Unsigned long_t.
 const long_t long_t_max = LONG_MAX;
 const long_t long_t_min = LONG_MIN;
 
-#define double_t_max ( DBL_MAX ) // because C++ language designers are apes
-#define double_t_min ( DBL_MIN ) // (only integral consts are compile time)
+/**
+ * only integral consts are compile time
+ */
```

Comment:
@SepehrMN Thanks for tidying up! But I think this should either be a proper Doxygen comment, e.g.
```
/**
 * Largest available double value.
 *
 * @note Compile-time constant must have integral type, thus we must define double limits as macros.
 */
```
or just the note text as a plain comment starting with `/*` instead of `/**`.